wrappers/nodejs/index.js

```diff
@@ -2060,6 +2060,7 @@ class Frame {
       case constants.format.FORMAT_GPIO_RAW:
       case constants.format.FORMAT_RAW10:
       case constants.format.FORMAT_FG:
+      case constants.format.RS2_FORMAT_Y411:
```

Comment:
The other ones don't have `RS2_`.... intended?

Reply:
fixed